scripts/repocheck.py

```diff
@@ -97,8 +97,7 @@
   'contact',
   'contact-email',
   'url',
-  'license',
-  'description',
```

Comment:
Cookiecutter pastes the project description into multiple files. This ensures that the module description is correctly set in the setup.py.